sys-process/runit/metadata.xml

```diff
@@ -12,4 +12,7 @@
   <upstream>
     <remote-id type="github">clan/runit</remote-id>
   </upstream>
+  <use>
+    <flag name="scripts">Install helper scripts using openrc, not enabled by default.</flag>
```

Comment:
It is enabled by default, so description is wrong. I don't think it makes sense to include this in description at all. Also, it's not exactly about "helper" scripts and not exactly about "openrc" scripts. Probably this one is better describe actual flag effect:
```suggestion
    <flag name="scripts">Install boot scripts into /etc/runit/.</flag>
```

Reply:
i mean helper scripts not enabled by default, confusion here, will change